### PR TITLE
Update acc api docs intsights

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [XSUP-17055](https://jira-hq.paloaltonetworks.local/browse/XSUP-17055)

## Description
This change changes to use the new ewndpoints and returns the new fields to the context

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/IntSight/Integrations/IntSight/IntSight.py [hunk 1]
```diff
@@ -838,7 +836,8 @@ def request_for_ioc_enrichment():
     response = http_request('GET', request_url, json_response=True)
     status = response.get('Status')
     if status == 'Done':
-        ioc_context, ioc_readable, dbot_score, domain, ip_info, url_info, hash_info = ioc_enrichment_to_readable(response)
+        ioc_context, ioc_readable, dbot_score, domain, ip_info, url_info, hash_info = ioc_enrichment_to_readable(
+            response)
```
Comment (JudahSchwartz) [inline Packs/IntSight/Integrations/IntSight/IntSight.py:840]:
```suggestion
        ioc_context, ioc_readable, dbot_score, domain, ip_info, url_info, hash_info = ioc_enrichment_to_readable(response)
```

**Comment by JudahSchwartz** [comment]
Needs force merge for the BC validation

File: Packs/IntSight/ReleaseNotes/2_0_0.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IntSights
+- ***Breaking Change:*** The *intsights-get-iocs* and *intsights-get-ioc-by-value* commands return different outputs according to the new api endpoints.
```
Comment (ShahafBenYakir) [inline Packs/IntSight/ReleaseNotes/2_0_0.md:4]: Let's specify the new API/product version
```suggestion
- ***Breaking Change:*** The *intsights-get-iocs* and *intsights-get-ioc-by-value* commands return different outputs according to the new API endpoints.
```

**Review by ShahafBenYakir — COMMENTED** [review]
- Could there be customers that are using the old API and it is still functioning for them?
- Let's add to the integration  README the API/product version we tested these commands

File: Packs/IntSight/ReleaseNotes/2_0_0.md (hunk 2 above)
Comment (JudahSchwartz) [reply Packs/IntSight/ReleaseNotes/2_0_0.md:4]: Intsights broke the api, which is where this bug came from. No reason to keep BC for the apis

**Comment by JudahSchwartz** [comment]
@ShahafBenYakir Intsights broke those apis so doesnt make sense to keep BC with them. Adding to the readme

**Pull request merged**